package.json

```diff
@@ -13,10 +13,10 @@
   "license": "ISC",
   "repository": "https://github.com/CLCInc/eslint-config-clc",
   "peerDependencies": {
-    "eslint": ">= 3"
+    "eslint": ">= 7"
   },
   "devDependencies": {
-    "eslint-config-prettier": "^6.5.0",
-    "eslint-plugin-prettier": "^3.1.1"
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-prettier": "^4.0.0"
```

Comment:
I went ahead and updated the dependencies and the minimum required eslint version thinking that as we update projects to use this package, we can also update the version of eslint they use. We have a mix of eslint versions across projects that import this package, with one as high as v7.30.0 so I think it's a safe upgrade. Eslint is actually at v8.8.0 right now, but I thought it best to keep the peer dependency lower incase we aren't able to update to that version.